### DISCUSSION TITLE
Animation when focus workspace by index

### DISCUSSION THREAD

**Discussion by dikesh** [YaLTeR/niri#661] [Q&A] [answered]
Is there any way to enable the animaton when focusing the workspace using index e.g. `focus-workspace 1` , `focus-workspace 3` etc like vertical scroll animation for workspace focus with `focus-workspace-up` / `focus-workspace-down`?

**Comment by YaLTeR** [comment, marked as answer]
It's already enabled on the latest git.

**Comment by jpeeler** [comment]
How about selectively disabling the animation? I kind of like it when using the default page up/down keys (focus-workspace). But I would rather the animations not be present when using focus-workspace-previous.

**Reply by YaLTeR** [reply]
Would it help if the animation didn't go through intermediate workspaces and went straight to the target one?

**Reply by jpeeler** [reply]
I believe so. But if you felt like implementing a property for instant switching as well (no-anim?), I'd probably use it.
